objectstorage/objectstorage.go

```diff
@@ -0,0 +1,123 @@
+package objectstorage
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/vocdoni/saas-backend/db"
+)
+
+type ObjectStorageConfig struct {
+	DB        *db.MongoStorage
+	ServerURL string
+}
+
+type ObjectStorageClient struct {
```

Comment:
I think we should remove any reference to the API or server urls from this package and move all that logic there. The API already has the route definitions and can encode and decode the resource URL in a more consistent way.